File: app/src/main/java/com/diwixis/filmlibrary/movies_module/MovieGreedView.java
package com.diwixis.filmlibrary.movies_module;

import com.diwixis.filmlibrary.structures.Result;

import java.util.List;

/**
 * Created by Diwixis on 19.04.2017.
 */

interface MovieGreedView {
    void showLoad();
    void hideLoad();
    void showMovie(List<Result> results);
}

File: app/src/main/java/com/diwixis/filmlibrary/movies_module/MovieGreedView.kt
package com.diwixis.filmlibrary.movies_module

import com.diwixis.filmlibrary.data.MoviesBean

/**
 * Created by Diwixis on 19.04.2017.
 */
interface MovieGreedView {
    fun showLoad()
    fun hideLoad()
    fun showMovie(movieList: List<Movie>)
}